dev/ci/ci-iris-lambda-rust.sh

```diff
@@ -34,8 +34,8 @@ git_checkout ${stdpp_CI_BRANCH} ${stdpp_URL_PARTS[0]} ${stdpp_CI_DIR} ${stdpp_UR
 # Build std++
 ( cd ${stdpp_CI_DIR} && make && make install )
 
-# Build iris
-( cd ${Iris_CI_DIR} && make && make install )
+# Build and validate iris
+( cd ${Iris_CI_DIR} && make && make validate && make install )
```

Comment:
Does this also validate std++? If not should we validate std++?

Reply:
It validates everything that's imported from std++, I think. If we also add `make validate` for std++, those parts would be validated twice.

Reply:
It looks like a defect of the `validate` target. It should use `coqchk -norec` not to recheck all dependencies (it even rechecks the stdlib today).

Reply:
I think this should be an option, but checking *all* dependencies also should be allowed.

`-norec` has to be passed in front of every file, right? It only applies to the next file? That's how I read the `-help` information, anyway.